### ISSUE TITLE
Integration with Home Assistant compromized after latest update

### ISSUE THREAD

**Issue by cavallaroe** [zigbee2mqtt/hassio-zigbee2mqtt#292]
**Description of problem:**
I have two roller shutter that are controlled by the Home Assistant through the zigbee2mqtt add on.
Yesterday I updated the HA to version 2022.2.6 and the integration with the zigbee devices stopped working.
I can still control them via the zigbee2mqtt GUI but they are not correctly seen by HA.
I tryed to remove and reinstall the zigbee2mqtt add on, as it solved some issue in the past, but it did not work this time.

The related issue on the HA forum is [here](https://community.home-assistant.io/t/mqtt-devices-not-working-on-lovelace-after-ha-update/392971)

**Zigbee2mqtt add-on version (if edge, please report commit hash):**
```
zigbee2mqtt@1.23.0-1
```

**Your entire configuration from the frontend (with sensitive fields redacted)**:
```
data_path: /config/zigbee2mqtt
external_converters: []
devices: devices.yaml
groups: groups.yaml
homeassistant: true
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
serial:
  port: /dev/ttyUSB0
advanced:
  log_level: warn
  pan_id: 6754
  channel: 11
  network_key:
    - 1
    - 3
    - 5
    - 7
    - 9
    - 11
    - 13
    - 15
    - 0
    - 2
    - 4
    - 6
    - 8
    - 10
    - 12
    - 13
  availability_blocklist: []
  availability_passlist: []
device_options: {}
blocklist: []
passlist: []
queue: {}
frontend:
  port: 8099
experimental: {}
availability: true
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: '-d -d'
  log: false

```

**Running platform (details from Suypervisor > System)**
```
// Example
Core: core-2022.2.6
Supervisor: supervisor-2022.01.1
Operating System: Home Assistant OS 7.4
```

**Your logs from Home Assistant Supervisor**
```
22-02-15 14:40:11 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.33.5:8099 ssl:default [Connect call failed ('172.30.33.5', 8099)]
22-02-15 14:40:12 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.33.5:8099 ssl:default [Connect call failed ('172.30.33.5', 8099)]
22-02-15 14:40:13 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.33.5:8099 ssl:default [Connect call failed ('172.30.33.5', 8099)]
22-02-15 14:40:14 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.33.5:8099 ssl:default [Connect call failed ('172.30.33.5', 8099)]
22-02-15 14:53:35 INFO (MainThread) [supervisor.misc.scheduler] Shutting down scheduled tasks
22-02-15 14:53:35 INFO (MainThread) [supervisor.api] Stopping API on 172.30.32.2
22-02-15 14:53:35 INFO (MainThread) [supervisor.hardware.monitor] Stopped Supervisor hardware monitor
22-02-15 14:53:35 INFO (MainThread) [supervisor.core] Supervisor is down - 100
22-02-15 14:53:35 ERROR (MainThread) [asyncio] Task was destroyed but it is pending!
task: <Task pending name='Task-79469' coro=<WSClient.start_listener() running at /usr/src/supervisor/supervisor/homeassistant/websocket.py:95> wait_for=<Future finished result=None>>
22-02-15 14:53:35 INFO (MainThread) [__main__] Closing Supervisor
22-02-15 13:53:45 INFO (MainThread) [__main__] Initializing Supervisor setup
22-02-15 13:53:46 INFO (MainThread) [supervisor.bootstrap] Initializing Supervisor Sentry
22-02-15 14:53:46 INFO (MainThread) [supervisor.bootstrap] Seting up coresys for machine: raspberrypi3-64
22-02-15 14:53:46 INFO (SyncWorker_0) [supervisor.docker.supervisor] Attaching to Supervisor homeassistant/aarch64-hassio-supervisor with version 2022.01.1
22-02-15 14:53:46 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
22-02-15 14:53:46 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.INITIALIZE
22-02-15 14:53:46 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-15 14:53:46 INFO (MainThread) [__main__] Setting up Supervisor
22-02-15 14:53:47 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2
22-02-15 14:53:47 INFO (MainThread) [supervisor.hardware.monitor] Started Supervisor hardware monitor
22-02-15 14:53:47 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface io.hass.os
22-02-15 14:53:47 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.systemd1
22-02-15 14:53:47 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.login1
22-02-15 14:53:48 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.hostname1
22-02-15 14:53:50 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.timedate1
22-02-15 14:53:51 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.NetworkManager
22-02-15 14:53:51 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface de.pengutronix.rauc
22-02-15 14:53:51 INFO (MainThread) [supervisor.host.info] Updating local host information
22-02-15 14:53:51 INFO (MainThread) [supervisor.host.services] Updating service information
22-02-15 14:53:52 INFO (MainThread) [supervisor.host.network] Updating local network information
22-02-15 14:53:53 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
22-02-15 14:53:53 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-02-15 14:53:53 INFO (MainThread) [supervisor.host.apparmor] Loading AppArmor Profiles: {'hassio-supervisor'}
22-02-15 14:53:53 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-cli with version 2021.12.0
22-02-15 14:53:53 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-dns with version 2021.06.0
22-02-15 14:53:53 INFO (MainThread) [supervisor.plugins.dns] Updated /etc/resolv.conf
22-02-15 14:53:53 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-audio with version 2021.07.0
22-02-15 14:53:53 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-observer with version 2021.10.0
22-02-15 14:53:53 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-multicast with version 2021.04.0
22-02-15 14:53:53 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
22-02-15 14:53:54 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/raspberrypi3-64-homeassistant with version 2022.2.6
22-02-15 14:53:54 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 1 Home Assistant secrets
22-02-15 14:53:54 INFO (MainThread) [supervisor.os.manager] Detect Home Assistant Operating System 7.4 / BootSlot A
22-02-15 14:53:56 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/a0d7b954 repository
22-02-15 14:53:56 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/core repository
22-02-15 14:53:56 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/45df7312 repository
22-02-15 14:53:56 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/cebe7a76 repository
22-02-15 14:53:58 INFO (MainThread) [supervisor.store] Loading add-ons from store: 65 all - 65 new - 0 remove
22-02-15 14:53:58 INFO (MainThread) [supervisor.addons] Found 7 installed add-ons
22-02-15 14:53:58 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to zigbee2mqtt/zigbee2mqtt-aarch64 with version 1.23.0-1
22-02-15 14:53:58 INFO (SyncWorker_3) [supervisor.docker.interface] Attaching to sabeechen/hassio-google-drive-backup-aarch64 with version 0.105.2
22-02-15 14:53:58 INFO (SyncWorker_5) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-configurator with version 5.3.3
22-02-15 14:53:58 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-mosquitto with version 6.0.1
22-02-15 14:53:58 INFO (SyncWorker_6) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-ssh with version 9.3.0
22-02-15 14:53:58 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-duckdns with version 1.14.0
22-02-15 14:53:58 INFO (SyncWorker_4) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/vscode/aarch64 with version 4.1.0
22-02-15 14:53:58 INFO (MainThread) [supervisor.backups.manager] Found 2 backup files
22-02-15 14:53:58 INFO (MainThread) [supervisor.discovery] Loaded 1 messages
22-02-15 14:53:58 INFO (MainThread) [supervisor.ingress] Loaded 63 ingress sessions
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-15 14:53:58 INFO (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-15 14:53:58 INFO (MainThread) [__main__] Running Supervisor
22-02-15 14:53:58 INFO (MainThread) [supervisor.os.manager] Rauc: A - marked slot kernel.0 as good
22-02-15 14:53:58 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons
22-02-15 14:53:58 INFO (MainThread) [supervisor.core] Supervisor reboot detected
22-02-15 14:53:58 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
22-02-15 14:53:58 INFO (MainThread) [supervisor.core] Supervisor is up and running
22-02-15 14:53:58 INFO (MainThread) [supervisor.host.info] Updating local host information
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.CORE
22-02-15 14:53:58 INFO (MainThread) [supervisor.host.services] Updating service information
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
22-02-15 14:53:58 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.PLUGIN
22-02-15 14:53:59 INFO (MainThread) [supervisor.host.network] Updating local network information
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-02-15 14:53:59 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
22-02-15 14:54:00 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-02-15 15:01:55 INFO (SyncWorker_3) [supervisor.docker.interface] Restarting ghcr.io/home-assistant/raspberrypi3-64-homeassistant
22-02-15 15:02:06 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
22-02-15 15:02:33 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-02-15 15:02:33 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-15 15:04:12 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
```

**Comment by tomekswx** [comment]
My network also started behaving strangely, it started with sensors, then the routers disappeared and now instead of zigbee2mqtt GUI I have 502 bad gateway.

**Comment by RaveGun** [comment]
Hi,

I just have a similar issue today. The Z2M devices were unavailable and the WebUI returned 502.
Additionally some other integrations that required an internet connection were also not running anymore, like HACS, Forecast.Solar etc.

I found out that there is an issue with the latest "hassio_dns" container on a not supported supervised installation. It stops running because of the apparmor that in Ubuntu (my case) is different than on Debian. There is already a fix in a "dev" container for the DNS (docker pull ghcr.io/home-assistant/amd64-hassio-dns:2022.03.dev0301).

So I ended up duplicating the original hassio_dns container but with the image source from above.

This did not fixed the problem directly because the /etc/resolv.conf was not pointing to the IP for the DNS container for both the homeassistant container and the z2m container.

After I changed the resolv.conf, all the missing devices and entities reappeared and the Z2M WebUI came back.

Maybe this helps somebody.

PS. I have a separate Portainer-CE installed on my machine (not, the now deprecated, AddOn from HA).
